### ISSUE TITLE
Import static pages from Github api

### ISSUE THREAD

**Issue by mattwr18** [AgileVentures/agile-ventures-website-react-front-end#46]
we need to bring in the site's static pages from the github api and convert them to html

helpful links:
https://github.com/octokit/rest.js/
https://octokit.github.io/rest.js/#api-Repos-getContents
https://github.com/rexxars/react-markdown

**Comment by mattwr18** [comment]
talking to @tansaku today in the Martin Fowler Scrum, we can just expose StaticPages api since we already have a nightly job, making this much easier to implement

**Comment by FedericoEsparza** [comment]
@mattwr18 I agree, exposing a StaticPages API is a more reliable and easier route.
We are once again guided in the right direction by @tansaku 🙂
